content/docs/1.0.0/tutorial/create-your-first-microservice-javascript.md

```diff
@@ -20,29 +20,21 @@ To keep things fairly simple, we'll want to have our microservice do something i
 
 So let's build a microservice that computes a real-time [moving average](https://wikipedia.org/wiki/Moving_average) from the input, sending the result to any other microservices that might be listening. We'll also set it up to use dynamic configuration for how long the rolling window should be for, so that we can change it later without needing to restart anything.
 
-If you're in a hurry, you can [skip ahead to the end if you'd like](#putting-them-together).
+If you're in a hurry, you can [skip ahead to the end if you'd like](#the-final-moving-average-code).
 
 ## Project Setup
 
 Since we're going to be writing a new microservice, we'll need to create a project directory. Let's create it inside our tutorial's previous working directory we should already be in.
```

Comment:
_Edits are bolded._

Let's create it inside our tutorial's previous working directory **that** we should already be in.